### DISCUSSION TITLE
Standard Error of Empirical Bayes Estimate

### DISCUSSION THREAD

**Discussion by EmanIbrahim137** [nlmixrdevelopment/nlmixr#599] [Q&A]
Hello,

I was wondering how can I get the standard error of the Empirical Bayes Estimates in nlmixr? similar to those obtained in the .phi file in NONMEM.

Thanks,
Eman

**Comment by mattfidler** [comment]
Hi Eman,

This not currently present in `nlmixr` so there is no way to get it now

**Comment by mattfidler** [comment]
It is now in one of the development branches of `nlmixr2`, so it will likely come out when generalized log likelihoods are implemented.

**Reply by EmanIbrahim137** [reply]
Looking forwards to it.
